terraform/pingdom/pingdom.tf

```diff
@@ -66,3 +66,18 @@ resource "pingdom_check" "cf_api_healthcheck" {
   notifywhenbackup         = true
   contactids               = ["${var.pingdom_contact_ids}"]
 }
+
+resource "pingdom_check" "paas_admin_healthcheck" {
+  type                     = "http"
+  name                     = "PaaS CF API - ${var.env}"
+  host                     = "admin.${var.system_dns_zone_name}"
+  url                      = "/calculator"
```

Comment:
Presumably `/calculator` so this covers a bit of the billing API too? (It does GET /pricing_plans I think) Or just because it's a page that doesn't need auth?

Reply:
Just because it doesn't need auth, although I was thinking of adding a health endpoint instead

Reply:
I think /calculator is fine for now. In a way it's nice that it covers billing a little bit, although that will make it a bit less reliable.